Claudia/src/main/java/com/endava/claudia/task/four/AdminUser.java

```diff
@@ -0,0 +1,23 @@
+package com.endava.claudia.task.four;
+
+public class AdminUser extends AbstractUser {
+
+    public AdminUser(String username, String addressType, Address address) {
+
+        super(username);
```

Comment:
Either call **super(username, addressType, address)** or remove this constructor as you already defined it bellow